stonesoup/movable/action/move_position_action.py

```diff
@@ -0,0 +1,89 @@
+import numpy as np
+from abc import abstractmethod
+from typing import Iterator, Sequence
+
+from ...base import Property
+from ...sensor.action import ActionGenerator, Action
+from ...types.state import StateVector
+
+
+class MovePositionAction(Action):
+    """This is the base class for an action that changes the
+    position of a platform or sensor."""
+
+    def act(self, current_time, timestamp, init_value):
+        return self.target_value
+
+
+class GridActionGenerator(ActionGenerator):
```

Comment:
Could we add a min and max method to the grid action generator (using the action space provided?) - this would allow it to be used with some of the optimised sensor managers we have.

Reply:
So would the min essentially return `current x position - n_steps*step_size*resolution` and max return `current x position + n_steps*step_size*resolution` ?

Reply:
I meant to remove the x, so it would be `current position +/- n_steps*step_size*resolution`